src/components/Personalization/dom-actions/scripts.js

```diff
@@ -80,8 +80,11 @@ export const getRemoteScriptsUrls = fragment => {
   return result;
 };
 
-export const executeInlineScripts = (container, scripts, func) => {
-  scripts.forEach(script => func(container, script));
+export const executeInlineScripts = (parent, scripts) => {
+  scripts.forEach(script => {
+    parent.appendChild(script);
+    parent.removeChild(script);
```

Comment:
How can you be sure the inline script is done executing? Do scripts that attach event handlers still work?